src/core/debug/shop-demo.js

```diff
@@ -21,6 +21,7 @@ class ShopDemo extends Screen {
 
 class ShopDemoGame extends Screen {
     create() {
+        this.plugins.addToScene(this.sys, [], [["ArcadePhysics"]]);
```

Comment:
ahh nice! I actually tried this but didn't realise that the 3rd arg needed to be an array of an array of a plugin (the docs don't mention that...)

Reply:
awful lot of magic string values in Phaser, huh.

Reply:
Yeah it's not an official way of doing things but I think ok for a debug screen. Not clear what's supposed to be private sometimes.